### PR TITLE
Property list

### PR DESCRIPTION
**Pull request by illiapelypenko**

### CODE REVIEW

File: src/components/PropertyList/PropertyMinified.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
```
Comment (uladzislau-stuk) [inline src/components/PropertyList/PropertyMinified.jsx]: PropertyList
PropertyCard

File: src/components/SearchResultsHeader/SearchResultsHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { ReactComponent as BtnLeftIcon } from '../../assets/icons/btnLeftIcon.svg';
+import { ReactComponent as BtnRightIcon } from '../../assets/icons/btnRightIcon.svg';
+import styles from './SearchResultsHeader.module.scss';
```
Comment (uladzislau-stuk) [inline src/components/SearchResultsHeader/SearchResultsHeader.jsx]: Think about better component name

File: src/containers/NothingFound/NothingFound.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+export function NothingFound() {
+  return (
+    <div className={styles.nothingFound}>
+      <img src={background} alt="background" />
+      <p>We are sorry! We have not found any properties.</p>
+      <Link to="/">Back to search</Link>
+    </div>
+  );
+}
```
Comment (uladzislau-stuk) [inline src/containers/NothingFound/NothingFound.jsx]: It's not container

File: src/utils/utils.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export function numberWithSpaces(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ' ');
+}
```
Comment (uladzislau-stuk) [inline src/utils/utils.js:3]: export function addSpacesTo(number) {}

File: src/routes/App.jsx [hunk 5]
```diff
@@ -1,24 +1,21 @@
...
         <Switch>
-          <Route exact path="/">
-            <Home />
-          </Route>
+          <Route exact path="/" component={Home} />
+          <Route path="/search" component={SearchResults} />
+          <Route path="/nothingfound" component={NothingFound} />
         </Switch>
       </div>
-    </Router>
```
Comment (uladzislau-stuk) [inline src/routes/App.jsx]: Rework navigator

File: src/containers/SearchResults/SearchResults.jsx [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+    offset: 0,
+  };
+
+  #SHOW_MORE_PROPS_VALUE = 20;
+
+  componentDidMount() {
+    const params = new URLSearchParams(this.props.location.search);
+
+    const city = params.get('city');
+    const state_code = params.get('state_code');
```
Comment (uladzislau-stuk) [inline src/containers/SearchResults/SearchResults.jsx:23]: use router api for getting params

Comment (illiapelypenko) [reply src/containers/SearchResults/SearchResults.jsx:23]: Can't find router API for getting params. (this.props.match.params is empty)

**Pull request merged**